qutebrowser/browser/commands.py

```diff
@@ -238,6 +247,23 @@ def tab_close(self, left=False, right=False, opposite=False, count=None):
             self._tabbed_browser.close_tab(tab)
             tabbar.setSelectionBehaviorOnRemove(old_selection_behavior)
 
+    @cmdutils.register(instance='command-dispatcher', scope='window', name='tab-pin')
+    @cmdutils.argument('index')
+    @cmdutils.argument('count', count=True)
+    def tab_pin(self, index=None, count=None):
```

Comment:
You'll need to add a docstring here, which is then used for the help and completion. This also makes the tests fail on Travis currently, as warnings are printed when there's a command without a docstring.